edge/pkg/clusterd/mission_deployer.go

```diff
@@ -106,7 +107,43 @@ func getDeleteContentCmd(mission *edgeclustersv1.Mission) (string, error) {
 	return resolveCommand(mission.Spec.MissionCommand.ReverseCommand), nil
 }
 
+//ApplyMission deploys missions based on given specifications
 func (m *MissionDeployer) ApplyMission(mission *edgeclustersv1.Mission) error {
+	configs := v1alpha1.Clusterd{Configs: v1alpha1.SetKubeconfigArray()}
+	if len(mission.Spec.Placement.Clusters) == 0 && len(mission.Spec.Placement.MatchLabels) == 0 {
+		for i := 0; i < len(configs.Configs); i++ {
+			config.Config.Kubeconfig = "/etc/fornax/configs/" + configs.Configs[i]
+			err := m.ApplyMissionCmd(mission)
+			if err != nil {
+				return err
+			}
+			if i == len(configs.Configs)-1 {
+				return nil
+			}
+		}
```

Comment:
add a return nil statement here and get rid of the "else"

Reply:
Made appropriate changes as per suggestions, thanks